x-pack/plugins/ml/common/util/string_utils.test.ts

```diff
@@ -46,4 +46,10 @@ describe('ML - string utils', () => {
       expect(result).toBe(0);
     });
   });
+
+  describe('stringHash', () => {
+    test('should return a number based off a string', () => {
+      expect(typeof stringHash('the-string')).toBe('number');
+    });
```

Comment:
There is no need in this test. TS covers it pretty well. But to assert the actual output would be great

Reply:
Updated the test in fbc9221. Rewrote it in a way to test what we're actually interested in: We want unique ids, but we don't care about the actual value.